gifted_build/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "webpack -v",
     "build": "webpack --config webpack-prod.config.js && webpack --config webpack-server.config.js",
-    "start": "node src/server/dev-server.js",
```

Comment:
Is there a reason you renamed these files? This might negate other work done on branches, but since not many others are working in the backend, this shouldn't affect too much, although naming references need to change.

Reply:
I had no real reason to rename the files besides making it a bit clearer to myself that `server.js` was the production server. I can undo the renaming, if it makes things easier/clearer.